ports/qtwebengine/portfile.cmake

```
@@ -226,7 +226,7 @@ qt_cmake_configure(
 )
 
 vcpkg_backup_env_variables(VARS PKG_CONFIG_PATH)
-file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Release" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Release/*/args.gn")
+file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Debug" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/core/Debug/*/args.gn")
```

This change is not suitable to fix anything.
This line collects `arg.gn` file paths, *relative to the base dir*. 
The original code assumes that there are the same relative paths in the debug build as in the release build. This is a fair assumption (until proven wrong).
The proposed code assumes that there are the same relative paths in the release build as in the debug build. This is bound to break release-only builds.

---

Indeed, I missed that. However, the build is not currently working when building both debug and release.

---

It is probably the right spot for a fix. But we must replace the `dbg`, not the `Release`.
```suggestion
file(GLOB target_args_gn RELATIVE "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/src/core/Release" "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/src/core/Release/*/args.gn")
```

---

> Will this work if you build only Debug ?

Debug-only is officially unsupported. Don't waste your time with it.